clang/lib/StaticAnalyzer/Checkers/StoreToImmutableChecker.cpp

```diff
@@ -11,6 +11,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "clang/AST/ParentMap.h"
```

Comment:
I don't think we need this include.

Reply:
Yeah, this was part of my attempt to use the structure of the AST to detect the lambda initialisation. Removed.